formats/definitions.json

```diff
@@ -151,7 +151,12 @@
             "format": "date-time"
           },
           "note": {
-            "type": "string"
+            "type": "string",
+            "description": "A summary of the change"
+          },
+          "full_description": {
+            "type": "string",
+            "description": "A more detailed description of the change. This is multiline text, and will be rendered inside <p> tags."
```

Comment:
What representation does multiline text use when this field is used?

Does it support Govspeak or is it just newline characters (`\n`) ?